### DISCUSSION TITLE
On release list pages, “Previous / Next” doesn’t make sense — should be “Newer / Older"

### DISCUSSION THREAD

**Discussion by hughlilly** [community/community#6181] [Other Feature Feedback, Questions, & Ideas] [answered]
On releases pages such as https://github.com/git/git/releases, this is the pagination navigation:
<img width="624" alt="image" src="https://user-images.githubusercontent.com/12046008/136688474-a0113062-963c-48ca-8676-d5d4044a8429.png">

While it is technically correct that the links go to the next page in the database, these labels read be “Newer” and “Older” respectively, since one is browsing the history of releases in chronological order.

Thanks
Hugh

**Comment by MylesBorins** [comment, marked as answer]
Previous / Next is a fairly common pattern for pagination. Often when paginating results the "next page" generally has chronologically older results. At this time this is not something we are looking to change for releases as we should be consistent with the rest of the platform (issues, prs, etc)

**Reply by hughlilly** [reply]
That's fine, but it doesn't make sense. Just because it's common doesn't mean it's good for usability.
